### ISSUE TITLE
Calculate weight tool does not work on Arcgis10.8

### ISSUE THREAD

**Issue by letter1225** [gtkfi/ArcSDM#139]
Hello,

The calculate weights tool  does not work on Arcgis10.8, as shown below.

![image](https://user-images.githubusercontent.com/29188553/96688755-1d353780-13b4-11eb-83f7-47227188a392.png)
The

**Comment by artolaiho** [comment]
Right-click on ArcSDM.pyt and select Check Syntax or move the cursor to the red X. Can you get more information.

**Comment by istone1922** [comment]
I also encountered the same problem,我的英语不好，主要问题是datetype有错误，检查语法，打开这个手册http://resources.arcgis.com/en/help/main/10.2/index.html#//001500000035000000进行更改

**Comment by fhgdgf** [comment]
I also encountered the problem_ Date type Property input value is invalid.
![mmexport1648313874845](https://user-images.githubusercontent.com/102422673/160249932-7f045de7-ab42-47bc-90c5-497bdd14ec1c.png)

**Comment by fhgdgf** [comment]
> Right-click on ArcSDM.pyt and select Check Syntax or move the cursor to the red X. Can you get more information.

> Right-click on ArcSDM.pyt and select Check Syntax or move the cursor to the red X. Can you get more information.

I also encountered the problem_ Date type Property input value is invalid
